pkg/nuget/packages_props/parse.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"encoding/xml"
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+	
+
+
+type propsPackageEntry struct{
+	Version         string   `xml:"Version,attr"`
+	UpdatePackageName     string   `xml:"Update,attr"`
+	IncludePackageName     string   `xml:"Include,attr"`
+}
+
+type propsPackageReferenceEntry struct {
+	XMLName         xml.Name `xml:"PackageReference"`
+	propsPackageEntry
+}
+
+type propsPackageVersionEntry struct {
+	XMLName         xml.Name `xml:"PackageVersion"`
+	propsPackageEntry
+}
+
+type propsItemGroup struct {
+	XMLName  xml.Name              `xml:"ItemGroup"`
+	ReferencePackages []propsPackageReferenceEntry `xml:"PackageReference"`
+	VersionPackages []propsPackageVersionEntry `xml:"PackageVersion"`
```

Comment:
looks like we can just use `propsPackageEntry`:
```suggestion
	ReferencePackages []propsPackageEntry `xml:"PackageReference"`
	VersionPackages   []propsPackageEntry `xml:"PackageVersion"`
```